library/ssl_tls13_keys.c

```diff
@@ -314,17 +314,14 @@ int mbedtls_ssl_tls1_3_evolve_secret(
     /* For non-initial runs, call Derive-Secret( ., "derived", "")
      * on the old secret. */
     if( secret_old != NULL )
-    {
-        ret = mbedtls_ssl_tls1_3_derive_secret(
-                   hash_alg,
-                   secret_old, hlen,
-                   MBEDTLS_SSL_TLS1_3_LBL_WITH_LEN( derived ),
-                   NULL, 0, /* context */
-                   MBEDTLS_SSL_TLS1_3_CONTEXT_UNHASHED,
-                   tmp_secret, hlen );
-        if( ret != 0 )
-            goto cleanup;
-    }
+        MBEDTLS_SSL_PROC_CHK(
```

Comment:
If a branch spans multiple lines, even if only belonging to a single statement, we prefer to keep the curly braces.